ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Add (MVI) Maximo Visual Inspection application to an existing MAS Core installation
+#
+# Dependencies:
+#  - ansible-playbook ibm.mas_devops.oneclick_core
+
+- hosts: localhost
+  any_errors_fatal: true
+
+  vars:
+    # Application Installation
+    # mas_app_id can be set to "health" to install manage in the "Health standalone" mode from this same playbook.
+    mas_app_id: "{{ lookup('env', 'MAS_APP_ID') | default('visualinspection', true) }}"
+    mas_app_channel: "{{ lookup('env', 'MAS_APP_CHANNEL') | default('8.5.x', true) }}"
+    mas_app_spec: "{'settings':{'storage':{'size':'100Gi','storageClassName':'ibmc-file-gold'}}}"
```

Comment:
@pseemung we cannot have a ROKS classname in a generic playbook like this, we need to do something similar to determine storage class automation used [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/db2/tasks/determine-storage-classes.yml) or [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/mongodb/tasks/determine-storage-classes.yml), so class can be automatically determined by the role that will use it, removing necessity of this line here in playbook. In summary:

- create a variable for the storage class (instead of the whole spec)
- add spec settings like in the default spec vars... look at how it is done in other applications.
- create a determine storage class automation yml similar the ones from db2 and mongo linked to you and be sure it is run only for MVI (as you probably need to do that in suite-app-install role, used for all the apps)
- recommend you to invite @caroazad and Sujala to a pair/trio programming session so you can exchange experience as well... let me know if you need any help here

Reply:
Changes are needed

Reply:
@pseemung @caroazad I still see the ROKS class being used in the playbook... and I don't see the changes suggested in my comment. Where is the script to auto detect storage class for mvi?